---a/net/minecraft/network/status/INetHandlerStatusServer.java
+++b/net/minecraft/network/status/INetHandlerStatusServer.java
@@ -4,10 +4,10 @@
 import net.minecraft.network.status.client.C00PacketServerQuery;
 import net.minecraft.network.status.client.C01PacketPing;
 
-public interface INetHandlerStatusServer extends INetHandler
-{
+public interface INetHandlerStatusServer extends INetHandler {
+
     void processPing(C01PacketPing packetIn);
 
     void processServerQuery(C00PacketServerQuery packetIn);
-}
+}
